Formula/siril.rb

```diff
@@ -29,7 +30,7 @@ class Siril < Formula
   depends_on "gnuplot"
   depends_on "gsl"
   depends_on "gtk+3"
-  depends_on "jpeg"
+  depends_on "jpeg-turbo"
```

Comment:
May want to wait on this until `opencv` migration.

Though, may not be an issue since `jpeg-turbo` uses v8 API while `jpeg` uses v9 API so it is rare to get linked incorrectly using common build systems like autotools, cmake, ... and we don't need to worry about Linux RPATH (e.g. like what happened for `luajit vs luajit-openresty`, `libomp vs llvm`)

The formulae that usually mess up linkage was from less common build systems like jam and upstream-maintained Makefiles.